backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCDSAbstractSuite.scala

```diff
@@ -208,12 +195,19 @@ abstract class GlutenClickHouseTPCDSAbstractSuite extends WholeStageTransformerS
     val df = spark.sql(Source.fromFile(new File(sqlFile), "UTF-8").mkString)
 
     if (compareResult) {
+      val fields = new util.ArrayList[StructField]()
+      for (elem <- df.schema) {
+        fields.add(
+          StructField
+            .apply(elem.name + fields.size().toString, elem.dataType, elem.nullable, elem.metadata))
```

Comment:
why add the fields.size?

Reply:
It's happend in q39a,q39b. Will accured error when set schema because they has same schema name